openhtf/util/validators.py

```diff
@@ -62,6 +62,14 @@ def MyPhase(test):
 
 _VALIDATORS = {}
 
+class ValidatorException()
+
+def wrap_call_exception(validator):
+  def wrapper(*args, **kwargs):
+    result = validator(args, kwargs)
+    if not result:
+      raise ValidatorException("%s conditions not met\nargs: %s kwargs: %s" % (validator.__name__, args, kwargs))
+    return result
```

Comment:
should this be returning the wrapped function?

Reply:
Yup...

Reply:
cooool.  and one more thing  :)   i think you also need asterisks for validator(*args, **kwargs).